cwms/__init__.py

```diff
@@ -7,6 +7,8 @@
 from .levels.location_levels import *
 from .levels.specified_levels import *
 from .locations.physical_locations import *
+from .outlets.outlets import *
```

Comment:
just need to update this to cwms.outlets... to match the new format of the init file.

Reply:
rebased and updated.